### DISCUSSION TITLE
Where/how to add my own functions so I can import them in the Python Script Widget

### DISCUSSION THREAD

**Discussion by meritwagner** [biolab/orange3#6493] [Q&A]
Hi everyone,

Might be a trivial thing but I just can not figure it out.
I created a py-file containing a function that helps me to fix decimal issues in german datasets (conversion between comma and dot) and I want to be able to just import this function in the Python scripting widget. I saved the Py-file in the Orange/data folder where also other packages, such as pandas_compat and such can be found. Still, I get the error, that my module can not be found.
Any ideas on this?

Thanks in advance,
Merit

**Comment by markotoplak** [comment]
Adding libraries into the Python environment of Orange makes them importable. The location depends on the OS and installation. The easiest way to find it is to run the following in a Python Script widget:

```
import site
print(site.getsitepackages())
```

This will output a list of folders. Putting scripts into any of them will make them importable. As an example, I get:

```
['/home/marko/venv310/lib/python3.10/site-packages', '/home/marko/venv310/local/lib/python3.10/dist-packages', '/home/marko/venv310/lib/python3/dist-packages', '/home/marko/venv310/lib/python3.10/dist-packages']
```
